docs/Development Concepts/7.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Experimental work in progress
+---
+
+The following features are currently experimental and under research and initial implementation:
+
+**Economic Spam resistance**:
+We aim to enable an incentivized spam protection technique to enhance `WAKU-RELAY` by using rate limiting nullifiers.
+More details on this can be found in [17/WAKU2-RLN-RELAY](https://rfc.vac.dev/spec/17).
+In this advanced method, peers are limited to a certain rate of messaging per epoch and an immediate financial penalty is enforced for spammers who break this rate.
+
+**Prevention of Denial of Service (DoS) and Node Incentivization**:
+Denial of service signifies the case where an adversarial node exhausts another node's service capacity (e.g., by making a large number of requests) and makes it unavailable to the rest of the system.
+DoS attack is to be mitigated through the accounting model as described in [18/WAKU2-SWAP](https://rfc.vac.dev/spec/18).
```

Comment:
Don't mention SWAP as we will research other protocols instead/first.

Reply:
@staheri14

What are your thoughts on DOS and service credentials? What can we say about this that is true and current?